src/components/molecules/Visualizer/Plugin/hooks.ts

```diff
@@ -61,6 +63,8 @@ export default function ({
   const popupVisible = useRef<boolean>(false);
   const externalRef = useRef<HTMLIFrameElement>(null);
 
+  const [uiVisible, onUIShow] = useState<boolean>(!!visible);
```

Comment:
`onUIShow` sounds like a function that is going to do something when the UI is shown. Is there a better name, like just `setUIVisibility` or `setUI` or something?

Reply:
Updated to `setUIVisibility`.